package.json

```diff
@@ -20,23 +20,21 @@
   },
   "devDependencies": {
     "@date-io/core": "^2.10.6",
```

Comment:
What is the need for date-io/core here in this package?

Reply:
@sirajalam049 resolve